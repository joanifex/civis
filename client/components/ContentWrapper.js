import React from 'react';

const ContentWrapper = ({children}) => (
  <div className='container'>
    {children}
  </div>
)

export default ContentWrapper;
