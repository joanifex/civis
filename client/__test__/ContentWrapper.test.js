import React from 'react';
import ContentWrapper from '../components/ContentWrapper.js';
import renderer from 'react-test-renderer';


test('Component is rendered', () => {
  const component = renderer.create(
      <ContentWrapper />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
