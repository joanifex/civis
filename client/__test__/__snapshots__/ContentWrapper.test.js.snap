// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component is rendered 1`] = `
<div
  className="container"
/>
`;
